pkg/features/isis/rpc.go

```diff
@@ -16,3 +18,45 @@ type adjacency struct {
 	Holdtime       int64  `xml:"holdtime"`
 	SNPA           string `xml:"snpa"`
 }
+
+type interfaces struct {
```

Comment:
You only need to have those attributes that you're actually interested in.

Reply:
What does it mean ? Should fields of struct interfaces that are no used be deleted?

Reply:
Done